Doctrine/Phpcr/StaticContent.php

```diff
@@ -11,14 +11,13 @@
 
 namespace Symfony\Cmf\Bundle\ContentBundle\Doctrine\Phpcr;
 
-use Doctrine\ODM\PHPCR\HierarchyInterface;
 use PHPCR\NodeInterface;
 use Symfony\Cmf\Bundle\ContentBundle\Model\StaticContent as ModelStaticContent;
 
 /**
  * PHPCR specific static content.
  */
-class StaticContent extends ModelStaticContent implements HierarchyInterface
```

Comment:
Yeah, problem is that we then should release Doctrine PHPCR-ODM 2.0. Or you should simply remove the deprecated methods from the interface, don't see anything wrong with that.